Cython/Compiler/Code.py

```diff
@@ -1145,6 +1145,14 @@ class GlobalState(object):
         'end'
     ]
 
+    h_code_layout = (
+        'h_code',
+        'utility_code_proto_before_types',
+        'type_declarations',
+        'utility_code_proto',
+        'end'
+    )
+
```

Comment:
This seems more like a list (with similar items) than a tuple (with items of different status or semantics).
```suggestion
    # h files can only have a much smaller list of sections
    h_code_layout = [
        'h_code',
        'utility_code_proto_before_types',
        'type_declarations',
        'utility_code_proto',
        'end'
    ]
```